### PR TITLE
Cli toggle mtls and config CA

### PR DESCRIPTION
**Pull request by mitchdraft**

### CODE REVIEW

File: cli/pkg/cmd/install/common.go [hunk 1]
```diff
@@ -56,10 +56,10 @@ func qualifyFlags(opts *options.Options) error {
 		}
 		//if iop.Mtls {
 		//	if iop.SecretRef.Name == "" {
-		//		return fmt.Errorf("please specify a secret name to use MTLS")
```
Comment (rickducott) [inline cli/pkg/cmd/install/common.go]: Why commented out? Should this be removed?

File: cli/pkg/cmd/meshtoolbox/mtls/root.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
+package mtls
+
+import (
+	"fmt"
+
+	"github.com/solo-io/solo-kit/pkg/api/v1/clients"
+	// "github.com/solo-io/solo-kit/pkg/api/v1/resources/core"
```
Comment (rickducott) [inline cli/pkg/cmd/meshtoolbox/mtls/root.go]: Remove

File: cli/pkg/cmd/meshtoolbox/policy/root.go [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+func Add(opts *options.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
+		Short: `apply a policy`,
+		Long:  `apply a policy`,
+		Run: func(c *cobra.Command, args []string) {
+			if err := addPolicy(opts); err != nil {
+				fmt.Println(err)
+				return
+				// return err
```
Comment (rickducott) [inline cli/pkg/cmd/meshtoolbox/policy/root.go]: Remove

File: cli/pkg/cmd/meshtoolbox/policy/root.go [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+func Remove(opts *options.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "remove",
+		Short: `remove a single policy`,
+		Long:  `remove a single policy`,
+		Run: func(c *cobra.Command, args []string) {
+			if err := removePolicy(opts); err != nil {
+				fmt.Println(err)
+				return
+				// return err
```
Comment (rickducott) [inline cli/pkg/cmd/meshtoolbox/policy/root.go]: Remove

File: cli/pkg/cmd/meshtoolbox/policy/root.go [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+func Clear(opts *options.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "clear",
+		Short: `clear all policies`,
+		Long:  `clear all policies`,
+		Run: func(c *cobra.Command, args []string) {
+			if err := clearPolicies(opts); err != nil {
+				fmt.Println(err)
+				return
+				// return err
```
Comment (rickducott) [inline cli/pkg/cmd/meshtoolbox/policy/root.go]: Remove

**Comment by rickducott** [comment]
Discussed in person. Going to put toggle mtls behind a flag pending some backend work to actually enable it, tracked in #95. The CA config should be in a good spot, but need to take one last look after #76 is merged in (this PR is on top of that, will get rebased to master)

**Comment by mitchdraft** [comment]
moved to #96

**Pull request closed without merging**